updates/2016-08-16.yml

```diff
@@ -1,3 +1,10 @@
+added:
```

Comment:
Hey @angelaLazar this is an old file, can you move the new content to a new `updates/2016-10-17.yml` file? Thanks.